src/data/index.ts

```diff
@@ -1,16 +1,16 @@
-const Values = Object.freeze({
+const Values = {
   api: {
     hostname: process.env.NODE_ENV !== 'production' ? 'localhost:1234' : 'https://toc-api.davwheat.dev/',
   },
-})
+} as const
 
-const Colors = Object.freeze({
-  toryPrimary: '#0087dc',
+const Colors = {
+  primary: '#0087dc',
   dark: '#203647',
   offWhite: '#fff',
-})
+} as const
```

Comment:
Oooo I never knew about, but I can immediately see why `as const` is useful! Thanks!